server/package.json

```diff
@@ -30,6 +30,7 @@
     "has": "^1.0.3",
     "helmet": "^4.2.0",
     "mongoose": "^5.10.15",
+    "request": "^2.88.2",
```

Comment:
Request library has been deprecated since Feb 2020 [Check official docu here](https://github.com/request/request#readme). I suggest using other libraries such as [Axios](https://www.npmjs.com/package/axios)